TcUnit-Runner/LaunchTcUnit.bat

```diff
@@ -100,6 +122,14 @@ IF "%~3" == "-a" (
 IF "%~3" == "-A" (
     SET TCUNIT_AMSNETID=%4
 )
+IF "%~3" == "-tc" (
+    SET TCUNIT_TCVERSION_TO_USE=%4
+)
+
+IF "%~3" == "-TC" (
+    SET TCUNIT_TCVERSION_TO_USE=%4
+)
+
 
```

Comment:
It's necessary to add ~5 and ~6 here as well. I'll fix it!

Reply:
An understanding question from me: Why only ~5 and ~6 ? what about ~2 and ~4 ?

Reply:
~2 and ~4 are there as well. This is just to handle the combinations of in which order you provide the arguments. I've already updated the script. I'll make sure to include my corrections into the script after merging your changes.